sdk/storage/Azure.Storage.Blobs/Changelog.txt

```diff
@@ -1,6 +1,14 @@
 Release History
 ===============
 
+12.0.0-preview.4 (2019-10)
+--------------------------
+- Added support for Customer Provided Key server side encryption
+- Verification of echoed client request IDs
+- Support for geo-redundant read from secondary location on failure
+- Added CreateIfNotExists and DeleteIfNotExists convenience methods
```

Comment:
Added CreateIfNotExists and DeleteIfNotExists convenience methods for Blobs

Reply:
Fixed - thanks!